paper/paper.md

```diff
@@ -32,82 +32,125 @@ affiliations:
-
-![checkout](workflow.jpg)
-
-If the researcher has access to non-neoplastic (*ie.* healthy) or another sort of 'control' tissue then **germline-filter** is the proper starting point. This module removes germline variants that are common between the control and the experimental tissue so as to not bias the results by including non disease causing variants. The user provides a 
-metadata file that indicates which experimental samples correspond to which control samples. We initiate a 
-thread pool then use the [vcfpy](https://pypi.org/project/vcfpy/) library to quickly identify shared variants across control/experimental matched vcf files, then write new ones that contain only unique variants. There is also the option to limit the reported variants to those found in NCBI's [dbSNP](https://www.ncbi.nlm.nih.gov/books/NBK21088/) and the Wellcome Sanger Institute's [COSMIC](https://cancer.sanger.ac.uk/cosmic) databases. This option is designed to give the user a higher
-degree of confidence in the pathogenic nature of each variant -- if independent experiments have reported a given variant, there is a higher likilihood that it is pathogenic. The output of **germline-filter** is a set of trimmed-down vcf files. 
-
-**count-mutations** builds a genome interval tree from the reference gtf, reads in a vcf file and converts it to a 
-[vcfpy](https://pypi.org/project/vcfpy/) object and then processes vcf entries in parallel. Each variant is matched to its
-corresponding gene and gene-wise counts are stored in shared memory. We then report the raw number of variants found in 
-each sample. The output is a csv file that contains counts for each sample versus every gene in the genome. 
-
-**find-aa-mutations** loads the reference gtf, indexes the genome fasta file with [pyfaidx](https://pypi.org/project/pyfaidx/) for memory efficient access and then constructes a genome interval tree 
-that can quickly match genomic coordinates from vcf entries to peptide-level mutations. In addition, if working 
-with cancer samples, the user has the option to filter out all mutations that are not found in the [COSMIC](https://cancer.sanger.ac.uk/cosmic) database and are therefore unlikely to be pathogenic. vcf files are split 
-chunk-wise and queried against the genome interval tree in parallel. Results are stored in a shared memory 
-object. We then convert variant hits to [Ensemble](https://uswest.ensembl.org/index.html) translation IDs, 
-in acordance to the [HGVS](https://varnomen.hgvs.org/) sequence variant nomenclature. The output is a 
-heirarchically ordered text file (csv or json) that reports the the Ensemble protein ID and the gene associated with 
-each variant, for each experimental sample. 
+associated with each observed variant. [OBB: Add a `head` output of an example vcf file]

```

Comment:
good idea

Reply:
hey im not sure how to do this bc VCF files are very wide (~150 char). if i try and add an example `head` half of it just gets cut off when i convert to a [JOSS pdf](https://whedon.theoj.org/). How can we get around this?

Reply:
i ended up artificially wrapping the record line and only displaying a single record. i think this looks ok

```
##fileformat=VCFv4.2
##source=HaplotypeCaller
#CHROM POS ID REF ALT QUAL FILTER INFO FORMAT
chr1 631391 . C T 72.28 . AC=2;AF=1.00;AN=2;DP=2;
        ExcessHet=3.0103;FS=0.000;MLEAC=1;MLEAF=0.500;MQ=NaN;
        QD=25.36;SOR=2.303 GT:AD:DP:GQ:PL 1/1:0,2:2:6:84,6,0
```